File: Programming Basics/02-Conditional Statements - Exercise/worldSwimmingRecord.js
function worldSwimmingRecord(input) {

    let worldRecord = Number(input[0]);
    let distance = Number(input[1]);
    let timeForOneMeter = Number(input[2]);

    let delay = Math.floor(distance / 15) * 12.5;
    
    let totalTime = distance * timeForOneMeter + delay;
    let difrence = Math.abs(worldRecord - totalTime);

    if(totalTime < worldRecord) {
        console.log(`Yes, he succeeded! The new world record is ${totalTime.toFixed(2)} seconds.`)
    } else {
        console.log(`No, he failed! He was ${difrence.toFixed(2)} seconds slower.`);
    }
    
}




worldSwimmingRecord(["55555.67",
"3017",
"5.03"]);

File: Programming Basics Java/src/ConditionalStatementsExercise/WorldSwimmingRecord.java
package ConditionalStatementsExercise;

import java.util.Scanner;

public class WorldSwimmingRecord {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        double worldRecordTime = Double.parseDouble(scanner.nextLine());
        double distance = Double.parseDouble(scanner.nextLine());
        double timeForOneMeter = Double.parseDouble(scanner.nextLine());

        double time = distance * timeForOneMeter;
        double additionalTime = Math.floor(distance / 15) * 12.5;
        double totalTime = time + additionalTime;
        double difference = Math.abs(worldRecordTime - totalTime);

        if (totalTime < worldRecordTime) {
            System.out.printf("Yes, he succeeded! The new world record is %.2f seconds.", totalTime);
        } else {
            System.out.printf("No, he failed! He was %.2f seconds slower.", difference);
        }
    }
}
